src/translations/en.json

```diff
@@ -2100,6 +2100,13 @@
               "yaml": "YAML file",
               "storage": "UI controlled"
             },
+            "dialog_new": {
+              "header": "Add dashboard",
+              "create_empty": "New dashboard from scratch",
+              "create_empty_description": "Start with an empty dashboard from scratch",
+              "default": "Overview dashboard",
```

Comment:
I would either go more generic or more specific:

- Call it "default dashboard" and we can upgrade it in the future to new defaults
- Call it "Masonry Dashboard" because it's tied to the layout engine that's used

Reply:
Let's go for default.